go.mod

```diff
@@ -0,0 +1,52 @@
+module github.com/Orange-OpenSource/cassandra-k8s-operator
+
+go 1.12
+
+require (
+	// cloud.google.com/go v0.41.0 // indirect
+	contrib.go.opencensus.io/exporter/ocagent v0.4.12 // indirect
+	github.com/coreos/etcd v3.3.10+incompatible
+	github.com/emicklei/go-restful v2.9.6+incompatible // indirect
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-openapi/spec v0.19.2 // indirect
+	github.com/go-openapi/swag v0.19.3 // indirect
+	github.com/gobuffalo/envy v1.7.0 // indirect
+	github.com/golang/groupcache v0.0.0-20190702054246-869f871628b6 // indirect
+	github.com/googleapis/gnostic v0.3.0 // indirect
+	github.com/gophercloud/gophercloud v0.2.0 // indirect
+	github.com/jarcoal/httpmock v1.0.4
+	github.com/kylelemons/godebug v1.1.0
+	github.com/mailru/easyjson v0.0.0-20190626092158-b2ccc519800e // indirect
+	github.com/operator-framework/operator-sdk v0.9.0
+	github.com/prometheus/common v0.6.0 // indirect
+	github.com/prometheus/procfs v0.0.3 // indirect
+	github.com/sirupsen/logrus v1.4.2
+	github.com/spf13/cobra v0.0.5 // indirect
+	github.com/stretchr/testify v1.3.0
+	github.com/swarvanusg/go_jolokia v0.0.0-20190213021437-3cd2b3fc4f36
+	github.com/thoas/go-funk v0.4.0
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/zap v1.10.0 // indirect
+	golang.org/x/crypto v0.0.0-20190701094942-4def268fd1a4 // indirect
+	golang.org/x/net v0.0.0-20190628185345-da137c7871d7 // indirect
+	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
+	golang.org/x/sys v0.0.0-20190626221950-04f50cda93cb // indirect
+	golang.org/x/tools v0.0.0-20190703212419-2214986f1668 // indirect
+	google.golang.org/genproto v0.0.0-20190701230453-710ae3a149df // indirect
+	google.golang.org/grpc v1.22.0 // indirect
+	k8s.io/api v0.0.0-20190612125737-db0771252981
+	k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236
+	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/klog v0.3.3 // indirect
+	k8s.io/kube-state-metrics v1.6.0
+	sigs.k8s.io/controller-runtime v0.1.12
+	sigs.k8s.io/controller-tools v0.1.11-0.20190411181648-9d55346c2bde // indirect
+)
+
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20190222213804-5cb15d344471
```

Comment:
why is that one needed when you could just change it in the requires ?

Reply:
I tried doing that, but the `replace` directive keeps getting added back. To be honest, I am not sure why as of yet. I have more focused on just getting things working and have not taken  much time to investigate.

Reply:
@jsanda when everything is ready let's try to take a new stab at it ;)